Lib/py_compile.py

```diff
@@ -76,7 +76,7 @@ def _get_default_invalidation_mode():
         return PycInvalidationMode.TIMESTAMP
 
 
-def compile(file, cfile=None, dfile=None, doraise=False, optimize=-1,
+def compile(file, cfile=None, dfile=None, doraise=False, quiet=0, optimize=-1,
```

Comment:
Since these aren't keyword-only arguments, users can invoke `compile()` like `compile('...', '...', '...', True, 1)`, so if we add `quiet` after `doraise`, we may break their code. It would be better if we could add `quiet` after `invalidation_mode`.